geoportal/c2cgeoportal_geoportal/scaffolds/create/env.default_tmpl

```diff
@@ -18,6 +18,8 @@ AUTHTKT_SAMESITE=Lax
 BASICAUTH=False
 GEOPORTAL_INTERNAL_URL=http://geoportal:8080
 MUTUALIZED_PRINT_URL=mutualized-print.apps.openshift-ch-1.camptocamp.com
+DISABLE_MUTUALIZED_PRINT=#
```

Comment:
Maybe a little comment here to explain the possible values ('' or '#')

Reply:
Have we not escape (or double) the `#` ?